Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -63,17 +65,38 @@ protected override void OnElementChanged(ElementChangedEventArgs<Editor> e)
 				Control.Started += OnStarted;
 				Control.Ended += OnEnded;
 				Control.ShouldChangeText += ShouldChangeText;
+				Control.ShouldBeginEditing += ShouldBeginEditing;
+				Control.ShouldEndEditing += ShouldEndEditing;
 			}
 
+			UpdateTextColor();
 			UpdateText();
 			UpdateFont();
-			UpdateTextColor();
 			UpdateKeyboard();
 			UpdateEditable();
 			UpdateTextAlignment();
 			UpdateMaxLength();
 		}
 
+		bool ShouldBeginEditing(UITextView textField)
+		{
+			if (textField.Text.Equals(Element.Placeholder))
```

Comment:
What happens if the placeholder is "Foo" and then the user types " Foo"?

Reply:
Right so I type "Foo" end exit.

Then I enter to start typing again and boop it deletes my text because my text is Foo and placeholder is Foo as well.

Reply:
@jassmith Is it will be okay if I add one more bool type variable to check if text is edited or not. Or do you have any other solution?

Reply:
Other solution use UILabel and [AddSubview](https://developer.xamarin.com/api/member/UIKit.UIView.AddSubview/p/UIKit.UIView/)

https://stackoverflow.com/questions/1328638/placeholder-in-uitextview